test/service_instance_and_binding_test/context_signature/suit_test.go

```diff
@@ -47,6 +47,10 @@ var _ = BeforeSuite(func() {
 		e.Set("api.osb_rsa_public_key", publicKeyStr)
 		e.Set("api.osb_successor_rsa_public_key", publicSuccessorKeyStr)
 	}).Build()
+	ctxSetup()
+})
+
+func ctxSetup() {
```

Comment:
this function does not setup the ctx, it registers a service broker. please rename it accordingly.